source/fundamentals/geo.txt

```diff
@@ -168,10 +168,9 @@ For more information on legacy coordinate pairs, see the
 Geospatial Indexes
 ------------------
 
-To enable querying for geosptial data, you must first create a supporting
+To enable querying for geospatial data, you must first create a supporting
```

Comment:
Suggestion:

The typo correction looks good, but I wonder if the "supporting index" could be made more clear. Perhaps something like this:

"To enable querying for geospatial data, you must create an index that corresponds to the type of data."

Also up for debate: whether "for" or "on" is the appropriate preposition in this case.

I don't have a strong opinion on this, but to me "querying for" means retrieving some data that is categorized as geospatial data and "querying on" is using that geospatial data as the criteria for the search.